easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'bx-python'
+version = '0.8.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/bxlab/bx-python'
+description = """The bx-python project is a Python library and associated set of scripts to allow for rapid
+ implementation of genome scale analyses."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('LZO', '2.10'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+prebuildopts = "export CPATH=$EBROOTLZO/include/lzo:$CPATH && "
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('python-lzo', '1.12', {
+        'modulename': 'lzo',
+        'checksums': ['97a8e46825e8f1abd84c2a3372bc09adae9745a5be5d3af2692cd850dac35345'],
+    }),
+    (name, version, {
+        'modulename': 'bx',
+        'checksums': ['ad0808ab19c007e8beebadc31827e0d7560ac0e935f1100fb8cc93607400bb47'],
+    }),
+]
+
```

@fizwit please add here:
```suggestion

sanity_pip_check = True

```

---

sanity_pip changes made.